framework/auth/core.py

```diff
@@ -843,6 +848,22 @@ def _get_unconfirmed_email_for_token(self, token):
 
         return verification['email']
 
+    def confirm_token(self, token):
```

Comment:
This is a bit of a misnomer: I would expect a method called `confirm_token` to return a confirm token, not validate it. How about `validate_token`?